src/vm/docs/mod.rs

```diff
@@ -241,6 +241,29 @@ Note: This function is only available starting with Stacks 2.1.",
 "#,
 };
 
+const IS_STANDARD_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(is-standard principal1)",
```

Comment:
Not that this is wrong, but I'm curious why this is `principal1` and not something more communicative, like `standard-or-contract-principal` or something?